src/jit/flowgraph.cpp

```diff
@@ -5726,33 +5749,14 @@ void Compiler::fgFindBasicBlocks()
 
     /* Now create the basic blocks */
 
-    fgMakeBasicBlocks(info.compCode, info.compILCodeSize, jumpTarget);
+    unsigned retBlocks = fgMakeBasicBlocks(info.compCode, info.compILCodeSize, jumpTarget);
 
     if (compIsForInlining())
     {
-        if (compInlineResult->IsFailure())
-        {
-            return;
-        }
-
```

Comment:
And I think the `CALLEE_DOES_NOT_RETURN` and `GTF_CALL_M_DOES_NOT_RETURN` are non-fatal so it would be triggered by the original reason in the following `compInlineResult->IsFailure()`